pkg/models/monitoring/sort_page.go

```diff
@@ -50,9 +50,22 @@ func (w wrapper) Less(i, j int) bool {
 	p := w.MetricValues[i]
 	q := w.MetricValues[j]
 
-	if p.Sample.Value() == q.Sample.Value() {
+	// Place Nil to the tail.
+	if p.Sample == nil && q.Sample != nil {
```

Comment:
nil < not nil : false
not nil < nil : true
You mean the nil should be placed at the end no matter it's descending or ascending order?

Reply:
Yes. Actually for a long time, the nil pointer panic has never been triggered, because Prometheus server response is always reliable and flawless.

See manually constructed faulty test data:
https://github.com/kubesphere/kubesphere/blob/14069b95f231440a9254b8ec5c700d397d90adf3/pkg/models/monitoring/testdata/faulty-node-metrics-sorted-2.json#L223-L227